### PR TITLE
Clean ap

### PR DESCRIPTION
**Pull request by Zerowhizkey**
Cleaned styles files "Mostly"

### CODE REVIEW

File: src/components/SliderItem/SliderItem.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { useNavigate } from 'react-router-dom';
 import { useRecipe } from '../../routes/Recipe/Recipe.context';
 import { SmallButton } from '../styles/Button.styles';
 import Heading from '../styles/Heading.styles';
+// import * as Styled from './SliderItem.styles';
```
Comment (linda-94-andersson) [inline src/components/SliderItem/SliderItem.tsx:6]: Should it be styled? And should the comment stay?

File: src/routes/Recipe/Recipe.styles.tsx [hunk 2]
```diff
@@ -1,75 +1,82 @@
 import { RiGroupFill, RiTimeFill } from 'react-icons/ri';
 import styled from 'styled-components';
```
Comment (linda-94-andersson) [inline src/routes/Recipe/Recipe.styles.tsx:2]: Change to styles? :)

File: src/routes/Recipe/Recipe.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { RiCheckboxBlankCircleLine, RiCheckboxCircleFill } from 'react-icons/ri';
+import { RiCheckboxBlankCircleLine } from 'react-icons/ri';
 import Heading from '../../components/styles/Heading.styles';
 import { useRecipe } from './Recipe.context';
 import * as Styled from './Recipe.styles';
```
Comment (linda-94-andersson) [inline src/routes/Recipe/Recipe.tsx:4]: Should be styles? :)

File: src/types/global.d.ts [hunk 4]
```diff
@@ -1,7 +1,6 @@
 interface ProviderProps {
     children?: React.ReactNode;
 }
-
```
Comment (linda-94-andersson) [inline src/types/global.d.ts:4]: Why remove spacing here? Or is it a formatting thing?

**Review by linda-94-andersson — CHANGES_REQUESTED** [review]
Please check the comments :)

File: src/routes/Recipe/Recipe.styles.tsx (hunk 2 above)
Comment (Zerowhizkey) [reply src/routes/Recipe/Recipe.styles.tsx:2]: not an issue! :D

File: src/routes/Recipe/Recipe.tsx (hunk 3 above)
Comment (Zerowhizkey) [reply src/routes/Recipe/Recipe.tsx:4]: nono

**Review by linda-94-andersson — APPROVED** [review]
Great job on cleaning! <3

**Pull request merged**